.github/workflows/ci.yaml

```
@@ -25,7 +25,7 @@ jobs:
         rust_version: [stable, "1.66"]
         include:
           - os: windows-2022
-            extra_args: "--exclude slint-node --exclude test-driver-nodejs --exclude ffmpeg"
+            extra_args: "--exclude test-driver-nodejs --exclude ffmpeg"
```

technicly we don't need to exclude the nodejs driver since it is already excluded once

```suggestion
            extra_args: "--exclude ffmpeg"
```